src/pages/WeatherForecast/components/Title.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react';
+import { styled } from '@mui/material/styles';
+
+import { dateFormat } from 'helpers/weather';
+
+const StyledTitle = styled('div')`
+  display: flex;
+  padding: 5px;
+  justify-content: space-between;
+  .day {
+    text-transform: uppercase;
+    font-size: 18px;
+  }
+  .date-and-month {
+    font-size: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: end;
+  }
+  .month {
+    font-size: 14px;
+  }
+  .date {
+    line-height: 1;
+    margin-bottom: 10px;
+  }
+`;
```

Comment:
could you pick one approach which you liked more (separate styled component like [src/containers/App/index.tsx](https://github.com/a-hnatiuk/weather/pull/1/files#diff-cb723f29fd8d3b0e8362eaac590a8052432a9f963ff5406056a76da970bde2cf) or classes inside like here) and apply it to whole project please?

Reply:
so, you decided to ignore the current comment?)

Reply:
no) I've changed all components. Reworked to separate styled component